config/default.json

```diff
@@ -1,6 +1,7 @@
 {
 	"WalletConfig": {
-		"LbryClaimAddress": "none"
+		"LbryClaimAddress": "none",
+		"DefaultChannel": "none"
```

Comment:
Great design here! Love that this makes it easy to use different channels on staging/dev and production as well as easy to change if forked.

Reply:
:+1: